app/src/main/java/comcmput301f16t01/github/carrier/Requests/DriverViewRequestActivity.java

```diff
@@ -239,6 +236,7 @@ public void getRoadAsync() {
      * This AsyncTask updates the road on the map and maps a route between two points. This is
      * so that it does not lock up the UI thread or try to make a network connection on it.
      */
+    // see code attribution
```

Comment:
"... on our wiki"
Maybe? haha. If you could find and replace that, that'd be perfect. 
But that's too tedious to do otherwise and it's fine as is.